code/game/objects/items/devices/radio/headset.dm

```diff
@@ -312,32 +312,8 @@
 
 /obj/item/radio/headset/attackby(obj/item/W as obj, mob/user as mob)
 	user.set_machine(src)
-	if(!( istype(W, /obj/item/screwdriver) || (istype(W, /obj/item/encryptionkey/ ))))
+	if(!istype(W, /obj/item/encryptionkey))

```

Comment:
This should be implemented as an `else return ..()`

Reply:
Both addressed, thanks!